src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import { User } from '../../types/User';
+import './UserInfo.scss';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { name, email } = user;
```

Comment:
You have different destructuring  styles. For example, here right in props:
```
export const CommentInfo: React.FC<Props> = ({
  comment: {
    name,
    email,
    body,
  },
})
```

I suggest to choose one. As for me, I prefer first variant: `const { name, email } = user;`